scripts/Makefile.build

```diff
@@ -181,6 +190,15 @@ $(obj)/%.o: $(src)/%.c $(objtool_obj) FORCE
 	$(call cmd,force_checksrc)
 	$(call if_changed_rule,cc_o_c)
 
+# Built-in and composite module parts
```

Comment:
I think this line is not needed here; it was probably copied accidentally.

Reply:
You are correct. :)